src/Behat/Behat/Snippet/Printer/ConsoleSnippetPrinter.php

```diff
@@ -84,4 +91,24 @@ public function printUndefinedSteps($suiteName, array $steps)
 
         $this->output->writeln('');
     }
+
+    /**
+     * @param array<string> $usedClasses
+     */
+    public function outputClassesUsesStatements(array $usedClasses): void
+    {
+        $classCount = \count($usedClasses);
```

Comment:
I prefer to use `if ($usedClasses === [])`, faster and shorter, feel free to ignore

Reply:
Oh! That's an oversight, I prefer this syntax too 😄